ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx

```diff
@@ -499,7 +498,8 @@ export const AttributesPage = withCurrentWorkspace() (
       const {node} = this.props;
       const {calculating, count, countError, form, loading, options} = this.state;
       const {formErrors, formValid} = this.validateForm();
-      const disabled = calculating || form.exists || !formValid || form.num.every(attr => attr.operator === 'ANY');
+      const disableAdd = calculating || !formValid || (this.isMeasurement && !formValid && !form.exists);
```

Comment:
Why is this snippet -> `(this.isMeasurement && !formValid && !form.exists)` not included in the function `validateForm()`? Seems like this is part of form validation

Reply:
Good catch. This is actually already checked in `validateForm()` now that I fixed the condition there, so I just removed this snippet and it works.